### ISSUE TITLE
Questions about how is Figure 3 drawn in the paper？

### ISSUE THREAD

**Issue by annmusic** [ts-kim/RevIN#3]
Thank you for making such a great contribution to the normalization！
I have some questions about Figure 3 in the paper. The proposed method  RevIn is to normalize the samples of a batch in the network, Does Figure 3(Effect of RevIN on distribution discrepancy between training and test data) depict the data distribution of the entire training set and test set? Here's how it's done.

**Comment by seharanul17** [comment]
Thank you for your interest in our work! Figure 3 illustrates the data distribution over the complete training and test sets. We aggregated all of the samples and visualized their distribution at each stage.
